src/components/UserBackgroundImage.js

```diff
@@ -36,17 +44,60 @@ const useStyles = makeStyles(() => ({
     zIndex: 'auto',
     backgroundColor: `rgba(0, 0, 0, 0.15)`,
   },
+  backgroundImageEnter: {
+    opacity: 0,
+  },
+  backgroundImageEnterActive: {
+    opacity: 1,
+    transition: 'opacity 2000ms',
+  },
+  backgroundImageAppear: {
+    opacity: 0,
+  },
+  backgroundImageAppearActive: {
+    opacity: 1,
+    transition: 'opacity 2000ms',
+  },
+  backgroundImageExit: {
+    opacity: 1,
+  },
+  backgroundImageExitActive: {
+    opacity: 0,
+    transition: 'opacity 2000ms',
+  },
 }))
 const UserBackgroundImage = ({ user }) => {
```

Comment:
There are a few major UX problems:
* The search bar is pushed off the page
* The fade appears to happen more than once, because it flickers. Is this a state/rendering problem?
* The fade is really jagged / not performant, at least for me on Firefox. Hard to investigate with the other stuff going on.

Reply:
Confirmed the flickering happens on Chrome, too, but the opacity transition performance is better there.